packages/swhid/swhid.0.2/opam

```diff
@@ -0,0 +1,57 @@
+opam-version: "2.0"
+synopsis: "OCaml library to work with Software Heritage identifiers"
+description:
+  "swhid is an OCaml library to work with persistent identifiers found in Software Heritage"
+maintainer: [
+  "Léo Andrès <contact@ndrs.fr>"
+  "Dario Pinto <dario.pinto@ocamlpro.com>"
+]
+authors: [
+  "Léo Andrès <contact@ndrs.fr>"
+  "Dario Pinto <dario.pinto@ocamlpro.com>"
+]
+license: "ISC"
+tags: [
+  "swh"
+  "software"
+  "heritage"
+  "archive"
+  "swhid"
+  "persistent"
+  "identifier"
+]
+homepage: "https://github.com/ocamlpro/swhid"
+bug-reports: "https://github.com/ocamlpro/swhid/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08"}
+  "digestif" {>= "1.0.0"}
+  "jsonm" {>= "1.0.1"}
+  "ezcurl" {>= "0.1"}
+  "bos" {>= "0.2.0"}
+  "fpath"
+  "swhid_core"
+  "bisect_ppx" {with-test & >= "2.6" & dev}
+  "odoc" {with-doc}
+]
```

Comment:
```
#=== ERROR while compiling swhid.0.2 ==========================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/OCamlPro/swhid/archive/refs/tags/0.2.tar.gz)
# path                 ~/.opam/4.08/.opam-switch/build/swhid.0.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p swhid -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/swhid-10091-d74c8e.env
# output-file          ~/.opam/log/swhid-10091-d74c8e.out
### output ###
# File "/home/opam/.opam/4.08/lib/eqaf/eqaf.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
#       ocamlc src/.swhid.objs/byte/swhid__Download.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.swhid.objs/byte -I src/.swhid.objs/public_cmi -I /home/opam/.opam/4.08/lib/astring -I /home/opam/.opam/4.08/lib/bigarray-compat -I /home/opam/.opam/4.08/lib/bos -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/curl -I /home/opam/.opam/4.08/lib/digestif -I /home/opam/.opam/4.08/lib/eqaf -I /home/opam/.opam/4.08/lib/ezcurl -I /home/opam/.opam/4.08/lib/ezcurl/core -I /home/opam/.opam/4.08/lib/fmt -I /home/opam/.opam/4.08/lib/fpath -I /home/opam/.opam/4.08/lib/jsonm -I /home/opam/.opam/4.08/lib/logs -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/rresult -I /home/opam/.opam/4.08/lib/swhid_core -I /home/opam/.opam/4.08/lib/uchar -I /home/opam/.opam/4.08/lib/uutf -intf-suffix .ml -no-alias-deps -open Swhid -o src/.swhid.objs/byte/swhid__Download.cmo -c -impl src/download.ml)
# File "src/download.ml", line 83, characters 8-19:
# 83 |         Result.bind (Swhid_core.Object.Hash.of_string dir) directory_unsafe )
#              ^^^^^^^^^^^
# Error: Unbound value Result.bind
#     ocamlopt src/.swhid.objs/native/swhid__Download.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlopt.opt -w -40 -g -I src/.swhid.objs/byte -I src/.swhid.objs/native -I src/.swhid.objs/public_cmi -I /home/opam/.opam/4.08/lib/astring -I /home/opam/.opam/4.08/lib/bigarray-compat -I /home/opam/.opam/4.08/lib/bos -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/curl -I /home/opam/.opam/4.08/lib/digestif -I /home/opam/.opam/4.08/lib/eqaf -I /home/opam/.opam/4.08/lib/ezcurl -I /home/opam/.opam/4.08/lib/ezcurl/core -I /home/opam/.opam/4.08/lib/fmt -I /home/opam/.opam/4.08/lib/fpath -I /home/opam/.opam/4.08/lib/jsonm -I /home/opam/.opam/4.08/lib/logs -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/rresult -I /home/opam/.opam/4.08/lib/swhid_core -I /home/opam/.opam/4.08/lib/uchar -I /home/opam/.opam/4.08/lib/uutf -intf-suffix .ml -no-alias-deps -open Swhid -o src/.swhid.objs/native/swhid__Download.cmx -c -impl src/download.ml)
# File "src/download.ml", line 83, characters 8-19:
# 83 |         Result.bind (Swhid_core.Object.Hash.of_string dir) directory_unsafe )
#              ^^^^^^^^^^^
# Error: Unbound value Result.bind
```
```suggestion
]
conflicts: [
  "result" {< "1.5"}
]
```